Kudu.Services/Deployment/OneDeployHelper.cs

```diff
@@ -5,22 +5,24 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using System.Text;
 
 namespace Kudu.Services.Deployment
 {
     static class OneDeployHelper
     {
-        private const string StackEnvVarName = "WEBSITE_STACK";
-
         // Stacks supported by OneDeploy 
         public const string Tomcat = "TOMCAT";
         public const string JavaSE = "JAVA";
         public const string JBossEap = "JBOSSEAP";
 
+        private const string StackEnvVarName = "WEBSITE_STACK";
+
         // All paths are relative to HOME directory
-        private const string ScriptsDirectoryRelativePath = "site/scripts";
-        private const string LibsDirectoryRelativePath = "site/libs";
+        public const string WwwrootDirectoryRelativePath = "site/wwwroot";

```

Comment:
NIT: I know this is a refactor but typically, `IEnvironment` has all these paths initialized as a singleton: `SiteRootPath`, `WebRootPath` etc. These paths could change as well.

Reply:
Good point. The same issue is there in Kudu and KuduLite. I'll leave it this way for now and bundle the improvement with the next changes in Kudu/KuduLite both.